src/gui/options_banlist.h

```diff
@@ -0,0 +1,29 @@
+#ifndef OPTIONS_BANLIST_H
+#define OPTIONS_BANLIST_H
+
+#include <QDialog>
+
+namespace Ui
+{
+    class options_banlist;
+}
+
+class options_banlist: public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit options_banlist(QWidget *parent = 0);
```

Comment:
Giving `0` by default is bad style IMO, because everyone just ignore it.
You should make use of it when you instantiate the class.